website/content/en/docs/concepts/deprovisioning.md

```diff
@@ -55,7 +55,7 @@ There are both automated and manual ways of deprovisioning nodes provisioned by
 * **Consolidation**: Karpenter works to actively reduce cluster cost by identifying when:
   * Nodes can be removed as their workloads will run on other nodes in the cluster.
   * Nodes can be replaced with cheaper variants due to a change in the workloads.
-* **Drift**: Karpenter will annotate nodes as drifted and deprovision nodes that have drifted from their desired specification. Currently, Karpenter will only automatically mark nodes as drifted in the case of a drifted AMI.
+* **Drift**: Karpenter will annotate nodes as drifted and deprovision nodes that have drifted from their desired specification. See [Drift](#drift) to see which fields are considered.
```

Comment:
Can this be a local reference (you can find other examples of how to do this with Netlify in the repo)? That way Netlify checks if there is a link failure due to a typo or something like that.